applications/webhooks/src/webhooks_shared_listener.erl

```diff
@@ -327,3 +332,15 @@ code_change(_OldVsn, State, _Extra) ->
 %%%===================================================================
 %%% Internal functions
 %%%===================================================================
+
+-spec should_handle_federated_events(binaries()) -> kz_proplist().
+should_handle_federated_events([]) -> [];
+should_handle_federated_events([H | T]) ->
+    case kz_util:to_binary(kz_nodes:local_zone()) == H of
+        'true' ->
+            [{<<"should_handle_federated">>, 'true'}];
+        _ when T == [] ->
+            [{<<"should_handle_federated">>, 'true'}];
```

Comment:
the only way that the case is not true is when T is not empty. Zones will be our and others, others may be empty but then our will be at the top.